client/state/breadcrumb/schema.js

```diff
@@ -0,0 +1,10 @@
+export const breadcrumbSchema = {
+	type: 'array',
+	items: {
+		type: 'object',
+		properties: {
+			label: { type: 'string' },
+			href: { type: 'string' },
+		},
+	},
+};
```

Comment:
This should be obsolete too if we drop ``withSchemaValidation`

Reply:
Deleted at https://github.com/Automattic/wp-calypso/pull/60491/commits/1814a84396098f3d617714d02c85292340a9de56